DEVELOPMENT_FLOW.md

```diff
@@ -0,0 +1,32 @@
+# OPI Project Development Flow
+
+This document attempts to document the development flow as the project moves
+towards code generation.
+
+## Sub-Projects and GitHub Repositories
+
+Each sub-project has their own GitHub Repository where they are currently
```

Comment:
we should say something about external public repos... guidelines... and maybe licensing...